project_script/main.py

```diff
@@ -29,49 +29,81 @@ def ask_file_name():
     return input("Insira o nome do arquivo para salvar \n")
 
 
-def ask_x_axys():
+def ask_x_axis():
     return input('Insira o cabeçalho do eixo x a ser considerado para o plote \n')
 
 
-def ask_y_axys():
+def ask_y_axis():
     return input('Insira o cabeçalho do eixo y a ser considerado para o plote \n')
 
 
-path_dark = ask_file_path("dark-CELIV")
+def is_bigger_data_frame(data_frame_1, data_frame_2):
+    return len(data_frame_1) < len(data_frame_2)
 
-while not is_valid_file(path_dark):
-    message_invalid_path()
-    path_dark = ask_file_path("dark-CELIV")
 
-dark_CELIV = pd.read_table(path_dark, sep='\t')
+def slice_data_frame_rows(data_frame_1, data_frame_2):
+    return data_frame_1.loc[0:len(data_frame_2)-1, :]
 
-current_dark_celiv_ramp_time = separate_even_columns(dark_CELIV).abs()
 
-path_photo = ask_file_path("photo_celiv")
+def sanitize_data_frames():
+    global even_columns_subtracted, time_photo_celiv_ramp_time
+    if is_bigger_data_frame(even_columns_subtracted, time_photo_celiv_ramp_time):
+        time_photo_celiv_ramp_time = slice_data_frame_rows(time_photo_celiv_ramp_time, even_columns_subtracted)
+        return
+    even_columns_subtracted = slice_data_frame_rows(even_columns_subtracted, time_photo_celiv_ramp_time)
 
-while not is_valid_file(path_photo):
-    message_invalid_path()
-    path_photo = ask_file_path("photo-celiv")
 
-current_photo_celiv_ramp_time = (separate_even_columns(pd.read_table(path_photo, sep='\t', header=None))).abs()
+def integrate_current_over_time():
+    global time_photo_celiv_ramp_time_transposed, even_columns_subtracted_transposed_in_arrays
+    integrated_values = []
+    for k, v in enumerate(even_columns_subtracted_transposed_in_arrays):
+        integrated_values.append(integrate.simps(even_columns_subtracted_transposed_in_arrays[k],
+                                                 time_photo_celiv_ramp_time_transposed[k], even='avg'))
```

Comment:
This indentation is made by Pycharm when I break the line. I don't know if it's a recommendation of PEP 8 (the standard followed in Python), but I'll see and let you know!

Reply:
Yeah, its a recommendation of PEP 8, in case of interest, u could check it out at the following link:

https://www.python.org/dev/peps/pep-0008/#indentation

Reply:
Even though, they let as an option to make the indentation the way you recommended, so, both are right according to the pattern documentation... This way, I can change it the way you suggested if you prefer to let the indentation in the breaks as the most common. Just let me know about it.

Reply:
In this case, let's follow the standard rules from the documentation. It's a little diferent from others, but, it's a good ideia to you, learn the specs from this language.